tests/qnn/test_qnn_torch.py

```diff
@@ -418,6 +418,34 @@ def test_gradients(self, get_circuit, n_qubits):
             assert torch.allclose(g1, g2)
         assert len(weights) == len(list(layer.parameters()))
 
+    @pytest.mark.parametrize("n_qubits, output_dim", indices_up_to(1))
+    def test_cuda_backward(self):
+        if not torch.cuda.is_available():
+            pytest.skip("Cuda device not available")
+        else:
+            try:
+                n_qubits = 4
+                dev = qml.device("default.qubit", wires=n_qubits)
+
+                @qml.qnode(dev, interface="torch")
+                def circuit(inputs, weights):
+                    qml.templates.AmplitudeEmbedding(inputs, wires=range(n_qubits), pad=0.0)
```

Comment:
```suggestion
                    qml.templates.AngleEmbedding(inputs, wires=range(n_qubits))
```

Reply:
Switched to AngleEmbedding as I was getting trouble with the numpy conversion in AmplitudeEmbedding